bin-src/__mocks__/previewStatsJson/preview-stats.trimmed.json

```diff
@@ -643,11 +643,11 @@
       ]
     },
     {
-      "id": "./node-src/ui/messages/errors/noCSFGlobs.stories.ts",
-      "name": "./node-src/ui/messages/errors/noCSFGlobs.stories.ts + 1 modules",
+      "id": "./node-src/ui/messages/errors/noCsfGlobs.stories.ts",
+      "name": "./node-src/ui/messages/errors/noCsfGlobs.stories.ts + 1 modules",
```

Comment:
Is this a valid rename in our rules?  `CSF` is an initialism, not a word.   Would we do `Tcp` instead of `TCP`?

Reply:
We also do `Dns` instead of `DNS` which feels weird.

Reply:
This is not a deal breaker for me.

Reply:
> Would we do `Tcp` instead of `TCP?`

Yes we would. 😞 

I honestly wasn't sure how I felt about it other than creating better separation in the filenames. I'd be down to flip them back though, WDYT?

Reply:
I think that's probably fine. I'll look into adding those exceptions and moving those files back!